### ISSUE TITLE
Support for custom Cross Validation strategies

### ISSUE THREAD

**Issue by AlexandruBurlacu** [vecxoz/vecstack#21]
The package looks amazing, but from what I saw, one can not pass a cross-validation sklearn object, only the number of folds, and enable/disable shuffling and stratification. This is an issue when trying to work with time series data, and using TimeSeriesSplit from sklearn. Would you consider adding maybe another toggle, like `time_series={True, False}` or even changing the API a bit, and instead of passing the number of folds and `shuffle` and `stratified` to have only one argument, like `cv` and pass a separate object from sklearn in there?

**Comment by AlexandruBurlacu** [comment]
Thank you for the suggested tutorial, and for explaining your reasons behind the more constrained cross-validation scheme. I think the issue should be closed then.

**Comment by Bturan19** [comment]
Hi,

I think implementing timeseriessplit is kind of strightforward too. 

As you mentioned, just the first fold will be filled with zeros.  This is just something we have to accept when try to stacking for time series models.
